indico/modules/events/tracks/templates/display.html

```diff
@@ -24,9 +24,15 @@
     <ul class="programme">
         {% for track in event.tracks %}
```

Comment:
If you login as someone who's not an admin, manager or global convener/reviewer I'm quite sure you are now sending at least one extra query for each track in the event.

To avoid this, you can do this in the controller and then use `tracks1 instead of `event.tracks` in the loop above.
```
        tracks = (Track.query
                  .with_parent(self.event_new)
                  .options(subqueryload('conveners'), subqueryload('reviewers')).
                  all())
```